source/meta.tex

```diff
@@ -2184,44 +2186,58 @@
 forms the logical conjunction of its template type arguments.
 
 \pnum
+For the specialization \tcode{disjunction<>},
+let $\tcode{B}_{i}$ be \tcode{true_type}.
```

Comment:
```suggestion
```
Based on the addition of the explicit specialization to the synopsis, and that only the primary template is present in the `itemdecl` environment, I believe this is not necessary.

Reply:
I agree we only need one or the other. The problem with specifying this via the synopsis is that it's easy to miss among dozens of other declarations. Specifying it here doesn't have that problem.